packages/marshal/src/marshal.js

```diff
@@ -34,16 +42,20 @@ const { details: X, quote: q } = assert;
 const QCLASS = '@qclass';
 export { QCLASS };
 
+/** @type {ConvertValToSlot<any>} */
 const defaultValToSlotFn = x => x;
+/** @type {ConvertSlotToVal<any>} */
 const defaultSlotToValFn = (x, _) => x;
 
 /**
  * @template Slot
- * @type {MakeMarshal<Slot>}
+ * @param {ConvertValToSlot<Slot>} [convertValToSlot]
+ * @param {ConvertSlotToVal<Slot>} [convertSlotToVal]
+ * @param {MakeMarshalOptions} [options]
  */
-export function makeMarshal(
-  convertValToSlot = defaultValToSlotFn,
-  convertSlotToVal = defaultSlotToValFn,
+export const makeMarshal = (
+  convertValToSlot = defaultSlotToValFn,
+  convertSlotToVal = defaultValToSlotFn,
```

Comment:
These got flipped (doesn't make much of a difference given their behavior).

Reply:
Good catch! Makes a huge difference, just not to the computer ;)

Reply:
I fixed this in https://github.com/endojs/endo/pull/1025/commits/b55a2bed166dbf8308a89d95a7c9ef2de41ee988